### PR TITLE
fix: og image url random parameter

### PR DESCRIPTION
**Pull request by kopancek**
## Changes

- Added random parameter to og image URL, so that when the image is changed, we get around the caching
- Use the same random parameter generation as we do in the API side.

As before, we can only test this in preview.

### CODE REVIEW

File: packages/webapp/components/layouts/ProfileLayout/index.tsx [hunk 1]
```diff
@@ -57,6 +57,7 @@ export default function ProfileLayout({
     process.env.NEXT_PUBLIC_API_URL,
   );
   ogImageUrl.searchParams.set('type', 'wide');
```
Comment (sshanzel) [inline packages/webapp/components/layouts/ProfileLayout/index.tsx:59]: Not part of this PR but wanted to ask, are we only going to use the wide version for OG?

**Review by sshanzel — APPROVED** [review]
Looks good to me 🚀

File: packages/webapp/components/layouts/ProfileLayout/index.tsx (hunk 1 above)
Comment (kopancek) [reply packages/webapp/components/layouts/ProfileLayout/index.tsx:59]: As far as I understand, yes: https://www.figma.com/file/nmfWPS7x3kzLUvYMkBx2kW/daily.dev---Dev-Mode?node-id=4234%3A17279&mode=dev
